python/cuspatial/pyproject.toml

```diff
@@ -134,6 +136,8 @@ requires = [
     "cmake>=3.26.4,!=3.30.0",
     "cudf==24.10.*,>=0.0.0a0",
     "cython>=3.0.0",
+    "libcudf==24.10.*,>=0.0.0a0",
```

Comment:
Putting this somewhere on the diff so it can be threaded.

Just realized I may have missed one detail that wouldn't be caught in CI.

@vyasr do we want something like https://github.com/rapidsai/cudf/blob/925530afe8178b7e788ea1a8d4df4c0eb4d042dc/python/cudf/cudf/__init__.py#L3-L11 near the top of `cuspatial`'s top-level `__init__.py`?

I *think* the answer is yes, so that other `libcudf.so` lying around in the environment aren't loaded first by some other import.

Reply:
Reading the linked code (and where it lives), it seems just doing `import cudf` would be sufficient

Though please let me know if I'm missing something

Reply:
John is right that the import should be sufficient for now. When you create the libcuspatial wheel in the follow-up, though, you'll end up needing to load libcudf there. [You can see how I did that in the cugraph PR for raft/cugraph-ops](https://github.com/rapidsai/cugraph/pull/4340/files#diff-c2e04ef6cef1505812981008cfadfa1901e4cfb9a537eee3cefaa3e93adaf9a8).